### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by Alusp**
## I implemented the following in this PR:

- [ ] Add Redux Toolkit(npm install react-redux @reduxjs/toolkit).
- [ ] Structure my application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files.
- [ ] configure my Redux Store and write actions and reducer for adding and removing books.
- [ ] Define action types for adding and removing a book.
- [ ] Set the initial state to be an empty array of books.
- [ ] Export Action Creators for your actions.
- [ ] Write your reducer and export it as default.
- [ ] Define state changes for the actions that you created.
- [ ] In case of unknown action - return the current state.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducer from './books/books';
```
Comment (Munsa1) [inline src/redux/configureStore.js]: - [x] `configureStore.js` should import `configureStore`  from @reduxjs/toolkit. Kindly fix it.
- [x] `configureStore.js` should import `categories` from `src/redux/categories`. Kindly fix it.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Alusp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly implement the following requirements for `categories.js`
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Alusp,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# highlights
- Descriptive pull request message ✔
approval
️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Alusp** [comment]
Thanks very much for the review @uwadonat

**Pull request merged**